src/templates/admin/submission/submit_funding.html

```diff
@@ -4,113 +4,129 @@
 {% load i18n %}
 {% load field %}
 
-{% block title-section %}{% trans "Article Info" %}{% endblock %}
+{% block title-section %}{% trans "Funding Information" %}{% endblock %}
 {% block css %}
-    <link href="{% static 'admin/css/timeline.css' %}" rel="stylesheet">{% endblock %}
+    <link href="{% static 'admin/css/timeline.css' %}"
+          rel="stylesheet">{% endblock %}
 
 {% block body %}
     <article>
         <div class="box">
-        {% include "submission/timeline.html" with data=article %}
-             <div class="title-area">
-                        <h2>{% trans "Funding" %}</h2>
+            {% include "submission/timeline.html" with data=article %}
+            {% if funder_form.errors %}
+                {% include "admin/elements/forms/errors.html" with form=funder_form %}
+            {% endif %}
+            <div class="title-area">
+                <h2>{% trans "Funding" %}</h2>
+
+            </div>
+            <div class="content">
+                <p>
+                    {% blocktrans %}
+                         Here you can search the FundRef database to add grant IDs to your article's metadata. If you can't find a specific funder, you can manually enter the details. You can also edit or delete entries as necessary.
+                    {% endblocktrans %}
+                </p>
+            </div>
+            <div class="title-area">
+                <h2>{% trans "Add Funding Source" %}</h2>
+            </div>
+            <div class="content submission-content">
+                <div class="row expanded">
+                    <div class="large-8 columns">
+                        <input type="text" name="funder_text"
+                               id="funder_text"/>
```

Comment:
So I know this is just registered as a change because you changed the indentation, but it would be good add a label for accessibility.